src/shared-components/Visual/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, LinearProgress } from '@mui/material';
+import { theme } from 'src/styles/theme';
+
+interface ProgressBarProps {
+  trackColor: string;
+  progressValue: number;
+  railColor: string;
+  height: number;
+}
+
+export const ProgressBar = ({
```

Comment:
Only ask for the `color` and `progressValue`. Set up other colors directly in the component. No need to be able to customize the height as we want a shared component that has consistent look when used across the app.

Reply:
Fixed